doc/using_the_computer_cluster.md

```diff
@@ -0,0 +1,7 @@
+# Using the computer cluster
+CECI (UCL cluster) mention

```

Comment:
Adding here more explanation of this otherwise I will forget :)
```suggestion
If you used the CECI (UCL cluster) for your project, mention it like this in the acknowledge section of your paper:

```